node/src/apis/lessons.ts

```diff
@@ -29,6 +29,7 @@ export interface Lesson {
   question: string;
   expectations: LessonExpectation[];
   conclusion: string[] | string;
+  createdBy: string;
```

Comment:
I think `dialog` doesn't use or need any of `createdAt`, `createdBy`, or `updatedAt`. 

Can we remove them all (all dummy values assigned around the src and tests too)?